eng/versioning/version_client.txt

```diff
@@ -110,23 +110,23 @@ com.azure:azure-messaging-servicebus;7.7.0;7.8.0-beta.1
 com.azure:azure-messaging-servicebus-track1-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-servicebus-track2-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-webpubsub;1.1.0;1.2.0-beta.1
-com.azure:azure-mixedreality-authentication;1.2.0;1.3.0-beta.1
-com.azure:azure-mixedreality-remoterendering;1.1.5;1.2.0-beta.1
+com.azure:azure-mixedreality-authentication;1.2.1;1.3.0-beta.1
+com.azure:azure-mixedreality-remoterendering;1.1.6;1.2.0-beta.1
 com.azure:azure-monitor-opentelemetry-exporter;1.0.0-beta.5;1.0.0-beta.6
 com.azure:azure-monitor-query;1.0.5;1.1.0-beta.1
 com.azure:azure-monitor-query-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-perf-test-parent;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
 com.azure:azure-search-documents;11.4.8;11.5.0-beta.9
 com.azure:azure-search-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-security-attestation;1.1.0;1.2.0-beta.1
+com.azure:azure-security-attestation;1.1.1;1.2.0-beta.1
 com.azure:azure-security-confidentialledger;1.0.0-beta.2;1.0.0-beta.3
-com.azure:azure-security-keyvault-administration;4.1.0;4.2.0-beta.1
-com.azure:azure-security-keyvault-certificates;4.3.0;4.4.0-beta.1
+com.azure:azure-security-keyvault-administration;4.1.1;4.2.0-beta.1
+com.azure:azure-security-keyvault-certificates;4.3.1;4.4.0-beta.1
 com.azure:azure-security-keyvault-jca;2.6.0;2.7.0-beta.1
 com.azure:azure-security-test-keyvault-jca;1.0.0;1.0.0
-com.azure:azure-security-keyvault-keys;4.4.0;4.5.0-beta.1
```

Comment:
Do we also want to update the latest stable version on the readme's too?

Reply:
Yes